.cfignore

```diff
@@ -9,3 +9,8 @@ cg-deck
 cg-dashboard
 /screenshots/
 static/tests/
+.org.chromium*
+# only exclude these index.html files from the VNC service.
+# they only appear in the root folder.
+/index.html*
```

Comment:
that's awesome! the great part is that i can put it in the container (which is linux) and it will clean up and then it will automatically reflect on the mac's disk